docs/jwst/error_propagation/main.rst

```diff
@@ -0,0 +1,374 @@
+'rateints' files, and updates the ERR arrays in those files. The details of the 
+calculations can be found at `Ramp fitting 
+<https://jwst-pipeline.readthedocs.io/en/latest/jwst/ramp_fitting/main.html>`_
+
+
+
+Group_scale
++++++++++++
+The group_scale step is applied after the ramp_fitting step, and applies to both the 
+rate and rateints products from ramp_fit. The correction is applied to the ERR, 
+(and will also be applied to the) VAR_POISSON, and VAR_RNOISE arrays.  The details
+of the calculations can be found at `Group Scale
+<https://jwst-pipeline.readthedocs.io/en/latest/jwst/group_scale/description.html>`_
+
+
+Stage 2 pipelines 
+-----------------
+Stage 2 pipelines perform additional instrument-level and observing-mode corrections and 
+calibrations to produce fully calibrated exposures. There are two Stage 2 pipelines: one 
+for `imaging 
+<https://jwst-pipeline.readthedocs.io/en/latest/jwst/pipeline/calwebb_image2.html>`_
+and one for `spectroscopy 
+<https://jwst-pipeline.readthedocs.io/en/latest/jwst/pipeline/calwebb_spec2.html>`_
+In these pipelines, the various steps that apply corrections and calibrations
+apply those same corrections/calibrations to all variance arrays arrays.
+
+There are two unique configuration files to be used to control these pipelines, 
+depending on whether the data are to be treated as a Time Series Observation 
+(in which case some of the steps are set to be skipped by default).
+
+
+Stage 2 Imaging Processing pipeline - calwebb_image2 & calwebb_tso-image2
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+The steps executed in this pipeline depend on whether the data are to be treated
+as TSO.  For non-TSO exposures, the steps are background image subtraction,
+assign_wcs, flat_field, photom, and resample.  For TSO exposures, the steps are
+assign_wcs, flat_field, and photom.
+
+Of the steps listed above, below are the steps that alter the ERR or VAR arrays
+of the science ramp data and output those altered arrays in the science product 
+that is passed to the next step in the pipeline.  For the steps not listed below,
+those arrays do not exist or are propagated unchanged to the next pipeline step.
+
+Flat_field
+~~~~~~~~~~
+(NOTE: get from the latest online readthedocs for this step once James updates it.)
+The SCI array from the flat-field reference file is divided into the VAR_POISSON
+and VAR_RNOISE arrays of the science data set. The variance in the flat is
+calculated in this step; the ERR array is then recalculated as these three 
+variances added in quadrature.  The newly created VAR_FLAT array and the updated 
+VAR_POISSON and VAR_RNOISE are output to the science products(s). For details of
+the flat fielding correction, see `Flatfielding
+<https://jwst-pipeline.readthedocs.io/en/latest/jwst/flatfield/index.html>`_
+
+
+Photom
+~~~~~~
+The photom step is applied for the modes: imaging and non-IFU spectroscopy,
```

Comment:
What's the point of listing "non-IFU spectroscopy" in the first entry, and then listing the 2 IFU modes in the second? I don't think this info as to how the step works is needed anyway, because it's all given in the photom step doc. Just delete this paragraph.